sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -153,8 +161,16 @@ Licensed under the MIT License.
       <artifactId>bcprov-jdk15on</artifactId>
       <version>1.68</version>  <!-- {x-version-update;org.bouncycastle:bcprov-jdk15on;external_dependency} -->
     </dependency>
-
-
+    <dependency>
+      <groupId>com.microsoft.data.encryption</groupId>
+      <artifactId>cryptography</artifactId>
+      <version>0.1.0.jre8-preview</version> <!-- {x-version-update;cosmos_com.microsoft.data.encryption:cryptography;external_dependency} -->
+    </dependency>
+    <dependency>
+      <groupId>com.microsoft.data.encryption</groupId>
+      <artifactId>azure-key-vault-keystoreprovider</artifactId>
+      <version>0.1.0.jre8-preview</version> <!-- {x-version-update;cosmos_com.microsoft.data.encryption:azure-key-vault-keystoreprovider;external_dependency} -->
```

Comment:
weird version. do they have different functionality for different versions of java?

Reply:
They have provided us 0.1.0.jre8-preview and 0.1.0.jre11-preview, as we support java 8 and onward, i picked this one.
In their pom they mentioned <finalName>${project.artifactId}-${project.version}.jre8${releaseExt}</finalName>